configs/next/distros/ubuntu-20.mk

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Distro dependent basic definitions for Ubuntu 20.04 LTS
+# =======================================================
+# This Makefile include contains the definitions for the distro being used
+# for the build process. It contains kernel version infos and distro specific
+# sanity checks.
+
+# Informs the distro supported power archs
+AT_SUPPORTED_ARCHS := ppc64le
+
+# You may force the BUILD_IGNORE_AT_COMPAT general condition by distro
+#BUILD_IGNORE_AT_COMPAT := yes
+
+# Kernel version to build toolchain against
+# - As this distro only supports one arch variation (ppc64le), there is no
+#   need to conditionally define these versions based on arch.
+AT_KERNEL := 5.4   # Current distro kernel version for runtime.
+# TODO: fill this when our packaging system starts supporting runtime-compat
+# on Ubuntu.
+#AT_OLD_KERNEL :=  # Previous distro kernel version for runtime-compat.
+
+# Inform the mainly supported distros
+AT_SUPPORTED_DISTROS := focal
+
+# Inform the compatibility supported distros
+#AT_COMPAT_DISTROS :=
+
+# Sign the repository and packages
+#AT_SIGN_PKGS := yes
+#AT_SIGN_REPO := yes
+#AT_SIGN_PKGS_CROSS := no
+#AT_SIGN_REPO_CROSS := yes
+
+# ID of GNUPG key used to sign our packages (main/compat)
+AT_GPG_KEYID := 6976A827
+AT_GPG_KEYIDC := 3052930D
+# ID of GNUPG key used to sign our repositories (main/compat)
+AT_GPG_REPO_KEYID := 3052930D
+AT_GPG_REPO_KEYIDC := 3052930D
+
+# Options required by the command to update the repository metadata
+AT_REPOCMD_OPTS := -p -s sha1 --simple-md-filenames --no-database
```

Comment:
I've seen ISOs being distributed with longer SHA.
Have you seen any movements in the DEB world to change this too?

Reply:
Answering my own question: it seems there was a transition indeed: http://archive.canonical.com/ubuntu/dists/focal/by-hash/
We should consider to move too.

Reply:
Updated `sha1` to `sha256`